django/contrib/postgres/fields/serial.py

```diff
@@ -0,0 +1,61 @@
+from django.db import connection
+from django.db.models import Field
+from django.utils.translation import ugettext_lazy as _
+
+
+class SerialField(Field):
+    description = _("Serial")
+
+    empty_strings_allowed = False
+    default_error_messages = {
+        'invalid': _("'%(value)s' value must be an integer."),
+    }
+
+    def __init__(self, *args, **kwargs):
+        kwargs['blank'] = True
+        kwargs.setdefault('unique', True)
+        super(SerialField, self).__init__(*args, **kwargs)
+
+    def deconstruct(self):
+        name, path, args, kwargs = super().deconstruct()
+        kwargs.pop('blank')
+        return name, path, args, kwargs
+
+    def validate(self, value, model_instance):
+        pass
```

Comment:
Hm... that's precisely what the `AutoField` does. You wouldn't have a value at this time, therefore there isn't much point in validating it, is there?